src/main/java/de/tum/in/www1/artemis/domain/notification/GroupNotification.java

```diff
@@ -9,10 +9,7 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.google.gson.JsonObject;
 
-import de.tum.in.www1.artemis.domain.Course;
-import de.tum.in.www1.artemis.domain.Exercise;
-import de.tum.in.www1.artemis.domain.Lecture;
-import de.tum.in.www1.artemis.domain.User;
+import de.tum.in.www1.artemis.domain.*;
```

Comment:
Codacy found an issue: [Avoid unused imports such as 'de.tum.in.www1.artemis.domain'](https://app.codacy.com/gh/ls1intum/Artemis/pullRequest?prid=5968854)